shell/common/engine.cc

```diff
@@ -248,15 +244,9 @@ void Engine::RunBundleAndSource(const std::string& bundle_path,
   if (packages_path.empty())
     packages_path = FindPackagesPath(main);
 
-  std::vector<uint8_t> platform_kernel;
-  if (!bundle_path.empty()) {
-    ConfigureAssetBundle(bundle_path);
```

Comment:
Wonder if this causes the same issue as https://github.com/flutter/engine/pull/4219. Seems to cause a regression https://github.com/flutter/flutter/issues/12739 that we've seen before.

Reply:
I'm sure it does! Sorry about that.

Reply:
https://github.com/flutter/engine/pull/4285 with the fix.